pypeit/scripts/ql_keck_mosfire.py

```diff
@@ -198,14 +269,50 @@ def main(args):
                                           merge_with=config_lines(args))
     science_path = os.path.join(parset['rdx']['redux_path'], parset['rdx']['scidir'])
 
-    # Calibration Master directory
-    if args.master_dir is None:
-        msgs.error("You need to set an Environmental variable MOSFIRE_MASTERS that points at the Master Calibs")
+    # Parse the files sort by MJD
+    files = np.array([os.path.join(args.full_rawpath, file) for file in args.files])
+    nfiles = len(files)
+    mjds = np.zeros(nfiles)
+    for ifile, file in enumerate(files):
+        hdr = fits.getheader(file, spectrograph.primary_hdrext)
+        try:
+           mjds[ifile] = hdr['MJD-OBS']
+        except:
+            msgs.warn('File {:} has not MJD in header'.format(file))
+            mjds[ifile] = 0
+    files = files[np.argsort(mjds)]
+
+    # We need the platescale
+    platescale = spectrograph.get_detector_par(None, 1)['platescale']
+    # Parse the offset information out of the headers. TODO in the future get this out of fitstable
+    dither_pattern, dither_id, offset_arcsec = parse_dither_pattern(files, spectrograph.primary_hdrext)
```

Comment:
Put this in `keck_mosfire.py` ?

Reply:
Done, and added dummy method to spectrograph.